src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -194,39 +198,40 @@ public ResponseEntity<GatewayCallbackResponse> onAddCareContext(
     return new ResponseEntity<>(HttpStatus.ACCEPTED);
   }
 
-  @PostMapping({"/v0.5/consents/hip/notify"})
+  //  @PostMapping({"/v0.5/consents/hip/notify"})
```

Comment:
Why is this API commented and why is a new function `initiateDataOnNotify` created?

Reply:
There was a discussion on which logic to be implemented, now the latest commit have the changes